public/pages/CreateTrigger/containers/ConfigureTriggers/ConfigureTriggers.js

```diff
@@ -217,7 +217,7 @@ class ConfigureTriggers extends React.Component {
   }
 
   async onQueryMappings() {
-    const indices = this.props.monitor.inputs[0].search.indices;
+    const indices = this.props.monitor.inputs[0].search?.indices;
```

Comment:
Quick clarification question. Do we need this to default to an empty list in situations where it's undefined?
E.g.,
```
const indices = this.props.monitor.inputs[0].search?.indices || [];

Reply:
addressed